controllers/appmesh/gatewayroute_controller.go

```diff
@@ -83,7 +89,11 @@ func (r *gatewayRouteReconciler) reconcile(ctx context.Context, req ctrl.Request
 	if !gr.DeletionTimestamp.IsZero() {
 		return r.cleanupGatewayRoute(ctx, gr)
```

Comment:
We should probably come back and add events here too.

Reply:
Good point, if deletion / cleanup fails we probably want events for that.

Reply:
Not a blocker, we can follow up on this.